common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1439,7 +1439,15 @@ def load_block_type(self, block_type):
         """
         Returns a subclass of :class:`.XBlock` that corresponds to the specified `block_type`.
         """
-        if block_type in self.disabled_xblock_types:
+        if hasattr(self, "_services") and 'request_cache' in self._services:
```

Comment:
Thanks for the suggestion-- it's (in my mind) much cleaner in the fetch_disabled_xblock_types method. Now I just have to figure out the test situation.

There is no Python style guideline for differentiating a method from an instance variable, correct? I've removed the "fetch" prefix now and gone back to simply disabled_xblock_types, although I feel uncomfortable about not indicating that this is a method instead a variable.